flocker/provision/_ssh/_conch.py

```diff
@@ -154,7 +153,8 @@ def perform_run_remotely(reactor, base_dispatcher, intent):
 
     def connect():
         connection = connection_helper.secureConnection()
-        connection.addErrback(lambda _: False)
+        connection.addErrback(write_failure)
```

Comment:
:grey_question: No action required, this relies on write_failure returning a Falsey value for retries (it returns None as of now so it's probably okay). It be sort of bad if someone changed eliot so that write_failure returned something else and suddenly we lost retries. That also seems pretty low risk to me though, so fine as is.